escapades_frontend/src/components/form/Signup.tsx

```diff
@@ -0,0 +1,110 @@
+import { useAuth } from "@/provider/authProvider";
+import { useNavigate } from "react-router-dom";
+import { SubmitHandler, useForm, FormProvider } from "react-hook-form";
+import classes from "./Form.module.scss";
+import axiosInstance from "@/api/axios";
+import { yupResolver } from "@hookform/resolvers/yup";
+import * as yup from "yup";
```

Comment:
I didn't know yup 👀

Reply:
very usefull..We used that at Faks. How do you handle validations ?

Reply:
I did manually, but this is nice!